numba/tests/test_npdatetime.py

```diff
@@ -385,12 +385,11 @@ def check(a, b, expected):
             expected_val = expected
             not_expected_val = not expected
 
-            if numpy_version >= (1, 16):
-                # since np 1.16 all NaT == comparisons are False, including
-                # NaT==NaT, conversely != is True
-                if np.isnat(a) or np.isnat(a):
-                    expected_val = False
-                    not_expected_val = True
+            # since np 1.16 all NaT == comparisons are False, including
```

Comment:
Multiple comments contrasting NP 1.16 with older behaviour (I won't add more comments on this file to save noise)